### PR TITLE
docs: remove crufty references to removed CFG_ENC_FS

### PR DESCRIPTION
**Pull request by lws-team**
Signed-off-by: Andy Green <andy@warmcat.com>

Re: https://github.com/OP-TEE/optee_os/issues/1259

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -385,7 +385,8 @@ Definitions:
 	*   Hisilicon HiKey Board, ARMv8-A (plat-hikey)
 *   AArch64 build of optee_os is now possible through the configuration `CFG_ARM64_core=y`
 *	Secure Storage: Data can be encrypted prior to their storage in the non-secure.
-	Build is configured using `CFG_ENC_FS=y`
+	Build is configured using `CFG_ENC_FS=y` (Note: this was later removed since
```
Comment (jbech-linaro) [inline CHANGELOG.md]: Wasn't `CFG_ENC_FS` still around in the 2.2.0 tag? IIRC this was removed in the internal 2.2.0 -> the coming 2.3.0?
```
$ git lg -1 && git grep CFG_ENC_FS | wc -l
* c0c5d39 - (HEAD, tag: 2.2.0, tag: m/master) Update CHANGELOG.md for 2.2.0 (3 months ago) <Jerome Forissier>
34
```
I.e, the CHANGELOG.md shouldn't be changed, the history is still intact. What we should do is add a note about it in for the 2.3.0 updates.

Comment (jforissier) [inline CHANGELOG.md]: @jbech-linaro true
```
$ git log --oneline 2.2.0..master --grep=CFG_ENC_FS
8dceff9 core: remove CFG_ENC_FS
7e1a7a6 travis: remove CFG_ENC_FS
```
Please don't touch `CHANGELOG.md`.

**Comment by jforissier** [comment]
@lws-team you need to add your `Signed-off-by:` to the commit message (this will fix the Travis/checkpatch failure)

**Comment by lws-team** [comment]
OK it should be updated accordingly.

**Comment by jforissier** [comment]
All good, thanks.

**Pull request merged**